packages/datepicker-react/documentation/Example.tsx

```diff
@@ -9,6 +9,8 @@ export const Example = ({ boolValues, choiceValues }: ExampleComponentProps) =>
     const errorLabel = boolValues && boolValues["Med feil"] ? "Du kan ikke velge en dato som har vært" : undefined;
     const variant = choiceValues && (choiceValues["Variant"] as LabelVariant);
 
+    const logDate = (message: string) => (date: Date | undefined) => console.log(message, date);
```

Comment:
For automatisk testing i fremtiden er det letter å bruke et eller annet element i dommet isteden for consolen for å representere endring, lettere å plukke opp.

Reply:
Ja, dette var egentlig mer for at jeg skulle se om det fungerte 😅 Vi bør helt klart prøve å finne et mønster på hvordan vi kan eksponere denne typen informasjon for test uten å kludre til eksempeler i portalen.